unittest.cfg

```diff
@@ -1,13 +1,19 @@
 [unittest]
 verbose = 2
-plugins = flufl.testing.nose
+plugins =
+    nose2.plugins.prof
```

Comment:
`flufl.testing.nose` conflicts with `nose2.plugins.prof`

Because IMHO profiling is more useful than style guide, I sacrificed the former for the latter.